### ISSUE TITLE
Name

### ISSUE THREAD

**Issue by maelle** [dmi3kno/bbox#1]
Just a note that @sckott has a package with the same name 🙂 https://github.com/ropensci/bbox

**Comment by dmi3kno** [comment]
Yeah, i saw it on available::available but decided to go ahead anyways. I am not sure I will proceed with the idea, but the hex sure looks nice. Maybe @skottie could use it?
